install/install.go

```diff
@@ -813,6 +816,10 @@ func (c *installer) createController(ctx *installerContext) error {
 		args = append(args, "--default-service-type="+c.commonOptions.DefaultServiceType)
 	}
 
+	if c.commonOptions.SkipLivenessProbes {
```

Comment:
is it not already applied in `configureAPIServerArgs()` ?

Reply:
I convinced myself those code paths didn't overlap - very possible I was wrong though!

Reply:
this one is only for the controller

Reply:
oh, right